src/aiida/tools/pytest_fixtures/daemon.py

```diff
@@ -56,6 +66,8 @@ def test(started_daemon_client):
             assert started_daemon_client.is_daemon_running
 
     """
+    if not daemon_client:
```

Comment:
TODO: I'll make this more robust and try to move this logic to `tests/conftest.py` to automatically detect these fixtures based on their names.